lib/storage/s3.js

```diff
@@ -53,10 +52,11 @@ module.exports = function() {
         options.params.httpOptions.agent = options.params.httpOptions.agent || options.agent;
       }
       client = new AWS.S3(options);
+      defaultTypes = require(__dirname + "/contentTypes.js");
       if (options.contentTypes) {
-        contentTypes = options.contentTypes;
+        _.extend(contentTypes, defaultTypes, options.contentTypes);
```

Comment:
You are extending contentTypes which is still null at this point I believe.

Reply:
This code doesn't seem to have anything to do with getUrl anymore...?

Reply:
I think this issue is already resolved in the existing codebase. I took too long, haha. Closing this.